pages/interface/components/TabCoinCount/index.js

```diff
@@ -9,7 +9,7 @@ export default function TabCoinCount({ amount, direction, mode = 'tooltip', sx }
     },
     full: {
       iconSize: 20,
-      text: `${amount?.toLocaleString('pt-BR')} TabCoin${amount !== 1 && 's'}`,
+      text: `${amount?.toLocaleString('pt-BR')} TabCoin${Math.abs(amount) === 1 ? '' : 's'}`,
```

Comment:
Considerem o que foi feito no #1461 como referência, então -1, 0 e 1 ficam no singular. Ficaria algo como:

```suggestion
      text: `${amount?.toLocaleString('pt-BR')} TabCoin${amount > 1 || amount < -1 ? 's' : ''}`,
```